solr/solr-ref-guide/modules/query-guide/pages/query-re-ranking.adoc

```diff
@@ -67,7 +67,7 @@ This number will be treated as a minimum, and may be increased internally automa
 |Optional |Default: `2.0`
 |===
 +
-A multiplicative factor that will be applied to the score from the reRankQuery for each of the top matching documents, before that score is added to the original score.
+A multiplicative factor that will be applied to the score from the reRankQuery for each of the top matching documents, before that score is combined with the original score.
 
 In the example below, the top 1000 documents matching the query "greetings" will be re-ranked using the query "(hi hello hey hiya)".
 The resulting scores for each of those 1000 documents will be 3 times their score from the "(hi hello hey hiya)", plus the score from the original "greetings" query:
```

Comment:
subjective: perhaps an additional example could be added where _"... plus the score ..."_ then is _"... times the score ..."_